src/main/resources/templates/db_scripts/schema.json

```diff
@@ -810,6 +810,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "effectiveLocationId",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
```

Comment:
There is no need to remove accents from an UUID. "removeAccents" should be false.

Reply:
changed.